ios/FluentUI/Vnext/Avatar/AvatarGroup.swift

```diff
@@ -0,0 +1,119 @@
+        HStack(spacing: 0) {
+            ForEach(0 ..< maxDisplayedAvatars, id: \.self) { index in
+                let modify = tokens.style == .stack && (overflowCount > 0 || index + 1 < maxDisplayedAvatars)
+                AvatarView(style: .default, size: tokens.size, state: avatars[index])
+                    .modifyIf(modify, { view in
+                        view.mask(AvatarCutout(xOrigin: x,
+                                               yOrigin: 0,
+                                               cutoutSize: size)
+                                    .fill(style: FillStyle(eoFill: true)))
+                    })
+                    .padding(.trailing, tokens.interspace)
+            }
+            if overflowCount > 0 {
+                createOverflow(count: Int(overflowCount), data: avatars.last)
+            }
+        }
+    }
+
+    private func createOverflow(count: Int, data: MSFAvatarState?) -> AvatarView {
+        data?.primaryText = "\(count)"
+        data?.image = nil
+        return AvatarView(style: .overflow, size: tokens.size, state: data)
+    }
+
+    /// Cutout shape for the succeeding Avatar in an Avatar Group in Stack style.
+    ///
+    /// xOrigin: beginning location of cutout on the x axis
+    ///
+    /// yOrigin: beginning location of cutout on the y axis
+    ///
+    /// cutoutSize: dimensions of cutout shape of the Avatar
+    private struct AvatarCutout: Shape {
+        var xOrigin: CGFloat
+        var yOrigin: CGFloat
+        var cutoutSize: CGFloat
+
+        func path(in rect: CGRect) -> Path {
+                var cutoutFrame = Rectangle().path(in: rect)
+                cutoutFrame.addPath(Circle().path(in: CGRect(x: xOrigin,
+                                                             y: yOrigin,
+                                                             width: cutoutSize,
+                                                             height: cutoutSize)))
+                return cutoutFrame
+        }
+    }
+}
+
+/// UIKit wrapper that exposes the SwiftUI AvatarGroup implementation
+@objc open class MSFAvatarGroup: NSObject, FluentUIWindowProvider {
+    @objc public init(style: MSFAvatarGroupStyle = .stack, size: MSFAvatarSize = .large, theme: FluentUIStyle? = nil) {
+        super.init()
+
+        avatarGroupView = AvatarGroup(style: style, size: size)
+        hostingController = UIHostingController(rootView: AnyView(avatarGroupView
+                                                                    .windowProvider(self)
+                                                                    .modifyIf(theme != nil, { personaView in
+                                                                        personaView.customTheme(theme!)
+                                                                    })))
+        view.backgroundColor = UIColor.clear
```

Comment:
It's safer to call the disableSafeAreaInsets here so we keep applying the workaround for scenarios where it's in the headers or anywhere close to the safe area limits.